client/src/js/app.js

```diff
@@ -67,6 +67,15 @@ function(Backbone, Marionette, _, $, HeaderView, SideBarView, DialogRegion, Logi
 
   app.parseQuery()
 
+  // Strip trailing forward slash sci-8800
+  if (location.href.endsWith('/')) {
+    location.href = location.href.replace(/\/+$/, '')
+  }
+
+  // Convert uppercase url to lowercase sci-8799
+  if (location.href.match(/[A-Z]/)) {
+    location.href = location.href.toLowerCase()
+  }
```

Comment:
Why not run these unconditionally?
```suggestion
  // Strip trailing forward slashes SCI-8800 and convert to lower case SCI-8799
  location.href = rtrim(location.href.toLowerCase(), "/")
```

Reply:
infinite redirect :)

Reply:
Fair 🙂